### PR TITLE
Add legacyApp field for slack

### PR DESCRIPTION
**Pull request by mrtc0**
For the integration with the Slack App to work, need to configure `slack.legacy-app` .

ref : https://github.com/getsentry/sentry/pull/19446

### CODE REVIEW

File: sentry/templates/configmap-sentry.yaml [hunk 1]
```diff
@@ -48,6 +48,9 @@ data:
     slack.client-id: "{{ .Values.slack.clientId }}"
     slack.client-secret: "{{ .Values.slack.clientSecret }}"
     slack.verification-token: "{{ .Values.slack.verificationToken }}"
+    {{- if .Values.slack.legacyApp }}
+    slack.legacy-app: "{{ .Values.slack.legacyApp }}"
```
Comment (Mokto) [inline sentry/templates/configmap-sentry.yaml]: Thanks !
From what I see it should be a "python" boolean, not a string. Have you tried the behaviour on your instance?

File: sentry/values.yaml [hunk 2]
```diff
@@ -192,6 +192,7 @@ slack: {}
 #   clientId:
 #   clientSecret:
 #   verificationToken:
+#   legacyApp:
```
Comment (Mokto) [inline sentry/values.yaml:199]: Should we put another comment to reference this issue ?
https://github.com/getsentry/sentry/pull/19446

File: sentry/templates/configmap-sentry.yaml (hunk 1 above)
Comment (mrtc0) [reply sentry/templates/configmap-sentry.yaml]: I haven't tried it in an instance. You're right, it should be a boolean here. I'll fix it.

File: sentry/values.yaml (hunk 2 above)
Comment (mrtc0) [reply sentry/values.yaml:199]: It's certainly better to be able to reference the issue.

File: sentry/templates/configmap-sentry.yaml (hunk 1 above)
Comment (Mokto) [reply sentry/templates/configmap-sentry.yaml]: If you put `#   legacyApp: true` in the values
and then

```

 {{- if .Values.slack.legacyApp }}
    slack.legacy-app: True
{{- else }}
    slack.legacy-app: False
```

That would make it easy to use

Comment (mrtc0) [reply sentry/templates/configmap-sentry.yaml]: Thanks for the good suggestions. fixed it.

**Comment by Mokto** [comment]
Thanks!

**Pull request merged**